src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.ts

```diff
@@ -327,4 +342,44 @@ export class TableInteractorColumnComponent implements OnChanges {
     }
     return null;
   }
+
+  interactorsTimesAppearing() {
```

Comment:
This method is no longer needed, we already increase `oneInteractor.timesAppearing` in line 372.